CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
```

Comment:
This is weird. Maybe you need a rebase. this file should not be part of the commit.